doc/examples/edges/plot_edge_filter.py

```diff
@@ -88,3 +88,73 @@
 
 plt.tight_layout()
 plt.show()
+
+######################################################################
+# As in the example above, we illustrate the rotational invariance
+# of the filters in this example. We see an rotationally invariant image
+# along with the angle of its analytical gradient in the top row.
+# The other two rows contain the difference between the different gradient
+# approximations (Sobel, Prewitt, Scharr & Farid) and the analytical gradient
+# Farid & Simoncelli derivative filters are most rotationally invariant, but
```

Comment:
Farid & Simoncelli derivative filters have the closest to rotation invariance, but require a 5x5 kernel which is computationally more intensive.